roles/openshift_node/tasks/upgrade/rpm_upgrade.yml

```diff
@@ -10,6 +10,8 @@
   command: "{{ ansible_pkg_mgr }} install -y --downloadonly {{ openshift_node_upgrade_rpm_list | join(' ')}}"
   register: result
   until: result is succeeded
+  failed_when:
+    - result.stdout is search(".*No package .* available.*")
```

Comment:
Add `or result is failed` ?
Basically we're working around the problem that `yum install a b` returns 0 if `a` is successful whether or not `b` fails. If they all fail for some other reason we should still fail. Not sure if failed_when is additive or absolute.